drivers/starburst/resources/metabase-plugin.yaml

```diff
@@ -12,6 +12,9 @@ driver:
   parent: sql-jdbc
   connection-properties:
     - host
+    - name: kerberos
```

Comment:
Does this need to be used anywhere? Does just adding the flag do anything?

Reply:
we are not using `kerberos` so we need to disable that

Reply:
not sure I understand, why do you need to add this to disable kerberos? How is it enabled be default?

Reply:
That's enabled by default

As a user I want to be able to disable that